.agent/workflows/update.md

```
@@ -0,0 +1,249 @@
+---
+description: Sync AGENTS.md, skill index, group workflows, and all.md from the filesystem — prevents drift and adapts to new repositories
+---
+
+> **Trigger**: After adding, removing, or renaming skills, or after copying `.agent` to a new repository.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)